adders/unplugin-icons/collections.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * This file is auto generated and can be updated by running:
```

Comment:
This is a very partial list, so I'm not sure how we'd decide which get listed here vs omitted. E.g. https://www.npmjs.com/package/@iconify-json/openmoji is missing. You can see all the icon sets at https://icon-sets.iconify.design/